### ISSUE TITLE
Power Source/Themes Confusing

### ISSUE THREAD

**Issue by cyanomys** [mlenser/kryx-rpg-issues#91]
So, we're creating characters and my players got SUUUUUPER confused by the wording concerning powersources on some of the classes:

![image](https://user-images.githubusercontent.com/63557988/80321452-b4914780-87e2-11ea-93aa-72f2d16d3f2f.png)

Basically, here are the range of readings that people did on that:
- some people got confused between themes and power sources, first of all
- some people thought that in order to learn feats or maneuvers, they had to be in BOTH their martial power source and chosen power source
- some people thought that they could only learn spells from their CHOSEN power source, and could learn spells, feats, and maneuvers from the Martial theme
- some people thought they could only learn feats and maneuvers from the martial theme, and spells, feats, and maneuvers from their chosen theme.

After about 45 minutes of confused discussion, this is what I ruled:
- You can learn feats and maneuvers from any theme you know, from either your chosen or martial powersource
- You can learn spells only from themes in your chosen power source, not themes in the martial power source.
- If a theme is in both your chosen and martial power source, you can learn spells from it.

Basically, the wording here needs to be clearer.  I can talk with my players about suggestions for clearer wording at some point (though right now we're back in the thick of character creation.)

**Comment by Marcloure** [comment]
Your ruling is correct. The idea is that, for instance, an Arcane half-caster shouldn't gain access to Beast or Earth spells just because those themes are also martial themes.

**Comment by mlenser** [comment]
You chose the most confusing class, huh? :P

I'll clean it up:

![image](https://user-images.githubusercontent.com/235857/80403764-70747480-88c0-11ea-98cc-6b4305b85bf4.png)
or a simpler version:
![image](https://user-images.githubusercontent.com/235857/80403784-79fddc80-88c0-11ea-8055-2d7c95a8a20c.png)


Please give suggestions if you think this wording can be further improved.

**Comment by mlenser** [comment]
Or even better:
![image](https://user-images.githubusercontent.com/235857/80404185-16c07a00-88c1-11ea-9295-ff5784a04329.png)
![image](https://user-images.githubusercontent.com/235857/80404209-1fb14b80-88c1-11ea-945f-295abd875e1f.png)

**Comment by mlenser** [comment]
> some people got confused between themes and power sources, first of all

If you have suggestions to clarify this let me know.

**Comment by cyanomys** [comment]
I've asked my players which ones read the most clearly to them and I will get back to you ^_^
